### ISSUE TITLE
Reference counter

### ISSUE THREAD

**Issue by akirataguchi115** [thunlp/TAADpapers#32]
It would be nice to see the reference amount on each paper. This could be picked from f.ex. scholar.google.com.

**Comment by yangalan123** [comment]
Good advice. May discuss it with team members. The consideration on our side is -- we do not really want to promote the atmosphere that the work with more citations is 100% better. Many things can happen behind the citation and not all of them are directly related to paper quality (also, we do not consider ourselves having the right the judge the paper quality. We rely on the peer-review system in conferences/journals to recommend papers.)

**Comment by akirataguchi115** [comment]
I hear you on the citations issue as it is really a problematic way to measure quality of a paper.

However some kind of order of the papers in this repo is already chosen. That is publication date which itself doesn't guarantee quality either (newer being better). So even if we didn't seemingly choose any order, we actually *are* choosing *some* order anyway.

**Comment by yangalan123** [comment]
True, we already give some ordering by giving chronological order. We will discuss whether to put citation numbers. Btw, can you draft a PR that adds the reference number? just to give us a sense how what would you expect it would look like -- you can only add reference numbers to a few papers.

**Comment by akirataguchi115** [comment]
Excellent idea. I'll draft a PR about this if I have time.
Otherwise this issue is a good invitation for anyone else to do that draft too.
